### ISSUE TITLE
Cursor keys and Ctrl keys not working

### ISSUE THREAD

**Issue by limtc** [ogoguel/activegs-ios#10]
Cannot use cursor keys or Ctrl key (says Ctrl-C to break Applesoft execution).

**Comment by OzCapital** [comment]
Yes - if you are running activegs on iPad/iPhone, Ctrl keys and arrows keys (and escape and "open" Apple key) do not work. This is because the iPad is capturing these characters and not passing them to the emulator. You need to mask the UITextField and process these characters before the iPad gets to consume them. I've done this is my version of activegs.
